mvt/feature.go

```diff
@@ -372,45 +198,12 @@ func (c *cursor) ClosePath() uint32 {
 
 // encodeGeometry will take a tegola.Geometry and encode it according to the
 // mapbox vector_tile spec.
-func encodeGeometry(ctx context.Context, geometry tegola.Geometry, tile *tegola.Tile, shouldSimplify bool, clip bool) (g []uint32, vtyp vectorTile.Tile_GeomType, err error) {
+func encodeGeometry(ctx context.Context, geometry tegola.Geometry) (g []uint32, vtyp vectorTile.Tile_GeomType, err error) {
 	if geometry == nil {
 		return nil, vectorTile.Tile_UNKNOWN, ErrNilGeometryType
 	}
 
-	c := NewCursor(tile)
-
-	// We are scaling separately, no need to scale in cursor.
-	c.DisableScaling = true
-
-	// TODO: gdey: We need to separate out the transform, simplification, and clipping from the encoding process. #224
-	geo := c.ScaleGeo(geometry)
```

Comment:
Potentially. I look at the scaling as essentially reprojecting the data. Since the data could theoretically be in a number of projections it seems to me that reprojecting the geometry to tile the desired tile coordinate system could be handled prior to passing to the encoder.

For the sake of this cutover, the scaling has to be handled in `atlas` since the validation routine only works well with coordinates in the tile projection.

Reply:
Yep. If you check the comment I have in atlas (where this was moved to) you will see additional notes.